test/functional/waivers_test.rb

```diff
@@ -10,6 +10,25 @@
   let(:controls_by_id)        { run_result; @json.dig("profiles", 0, "controls").map { |c| [c["id"], c] }.to_h }
   let(:cmd)                   { "exec #{waivers_profiles_path}/#{profile_name} --input-file #{waivers_profiles_path}/#{profile_name}/files/#{waiver_file}" }
 
+  attr_accessor :out
+
+  def inspec(commandline, prefix = nil)
+    @stdout = @stderr = nil
+    self.out = super
+  end
+
+  def stdout
+    @stdout ||= out.stdout
+      .force_encoding(Encoding::UTF_8)
+      .gsub(/\e\[(\d+)(;\d+)*m/, "") # strip ANSI color codes
```

Comment:
There's a `--no-color` option you can pass if you don't want color characters

Reply:
Nice! I was just using some setup code like it had been used in `inspec_exec_test`; might be worth extracting and unifying at some point but I didn't want to go too far with refactoring in this issue